scripts/pre_ingest_sanitisation.py

```diff
@@ -0,0 +1,236 @@
+# For an annotated version, use the ipython notebook.
```

Comment:
I'm worried about keeping things in sync.  If this is not auto generated from the ipynb, it could easily drift away from what is in there.